apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2

```diff
@@ -1699,6 +1699,7 @@
         {% endif %}
         "isSignatureValidationCertificateAliasEnabled": {{ console.applications.ui.certificate_alias_enabled }},
         "listAllAttributeDialects": {{ console.list_all_attribute_dialects }},
+        "excludeIdentityClaims": {{ console.exclude_identity_claims }},
```

Comment:
Add a null check for the config as well.

Reply:
Else, there'll be a JSON parse error if the framework and console versions don't sync.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/5350/commits/7d3ce4d4b2ae15058599ef21dfd976f4b8f54974